README.md

```diff
@@ -2,616 +2,398 @@
+| `disabled$`             | `Observable<boolean>`                                                       | Required             | ✅        | ❌       | When this observable emits `true`, the whole form (including the root form and all the sub forms) will be disabled                                                                                                                                                                                                                                                                                               |
+| `input$`                | `Observable<ControlInterface \| undefined>`                                 | Required             | ✅        | ❌       | A root form is a component in between the parent passing raw data and the form itself. This property is an observable that you must provide which will be used behind the scenes to update for you the form values                                                                                                                                                                                               |
+| `output$`               | `Subject<ControlInterface>`                                                 | Required             | ✅        | ❌       | A root form is a component in between the parent passing raw data and the form itself. This property is an observable that you must provide which will be used behind the scenes to broadcast the form value to the parent when it changes                                                                                                                                                                       |
+| `manualSave$`           | `Observable<void>`                                                          | Optional             | ✅        | ❌       | By default a root form will automatically broadcast all the form updates (through the `output$`) as soon as there's a change. If you wish to "save" the form only when you click on a save button for example, you can create a subject on your side and pass it here. Whenever you call `next` on your subject, assuming the form is valid, it'll broadcast te form value to the parent (through the `output$`) |
+| `outputFilterPredicate` | `(currentInputValue: FormInterface, outputValue: FormInterface) => boolean` | Optional             | ✅        | ❌       | The default behaviour is to compare the current transformed value of `input$` with the current value of the form _(deep check)_, and if these are equals, the value won't be passed to `output$` in order to prevent the broadcasting                                                                                                                                                                            |
```

Comment:
```suggestion
| `outputFilterPredicate` | `(currentInputValue: FormInterface, outputValue: FormInterface) => boolean` | Optional             | ✅        | ❌       | The default behaviour is to compare the current transformed value of `input$` with the current value of the form _(deep check)_, and if these are equal, the value won't be passed to `output$` in order to prevent the broadcast                                                                                                                                                                            |
```